CHANGES.md

```diff
@@ -1,12 +1,144 @@
+- [#4126](https://github.com/gap-system/gap/pull/4126) Added a command line option `--version`
+- [#4128](https://github.com/gap-system/gap/pull/4128) Added `OutputGzipFile`, in order to create a gzip compressed file
+- [#4132](https://github.com/gap-system/gap/pull/4132) Fixed a long standing bug which prevented `SetPrintFormattingStatus( "*stdout*", false );` from working as expected
+- [#4142](https://github.com/gap-system/gap/pull/4142) Extend `BindConstant`, `MakeConstantGlobal` to all object types
+- [#4145](https://github.com/gap-system/gap/pull/4145) Improved some errors messages for various set operations (`AddSet`, `UniteSet`, ...)
+- [#4146](https://github.com/gap-system/gap/pull/4146) Fixed `Cite` w.r.t. the encoding used; let `BibEntry` always use encoding "UTF-8"
+- [#4168](https://github.com/gap-system/gap/pull/4168) Improved the documentation concerning GASMAN, added `CollectGarbage`
+- [#4186](https://github.com/gap-system/gap/pull/4186) Improved the performance of `CoeffientsQadic` for long results
+- [#4201](https://github.com/gap-system/gap/pull/4201) Support an optional `transformFunction` in `Test()`, similarto `compareFunction`
+- [#4206](https://github.com/gap-system/gap/pull/4206) Fixed `DirectoriesPackageLibrary`, `DirectoriesPackagePrograms` when they are called during package loading
+- [#4207](https://github.com/gap-system/gap/pull/4207) Added the group constructors `PGammaL`, `PSigmaL`, and their methods for permutation groups
+- [#4215](https://github.com/gap-system/gap/pull/4215) `libgap` API: Expose a minimal interface to the garbage collector, via `GAP_MarkBag`, `GAP_CollectBags`
+- [#4219](https://github.com/gap-system/gap/pull/4219) Fixed `IntermediateGroup`: For large indices, it might have returned `fail` although subgroups exist, instead of issuing an error message or warning about a hard calculation
+- [#4232](https://github.com/gap-system/gap/pull/4232) Iterators for lists: fixed `IsDoneIterator`, improved the performance
+- [#4239](https://github.com/gap-system/gap/pull/4239) Added missing comparison w.r.t. equality of something and an object with memory (`IsObjWithMemory`)
+- [#4245](https://github.com/gap-system/gap/pull/4245) Free (associative or Lie) algebras: admit non-fields as left acting domains, fixed the zero-dimensional case
+
+-->
+
+## GAP 4.11.1 (February 2021)
 
 These changes are also listed on the
 [Wiki page](https://github.com/gap-system/GAP/wiki/gap-4.11-release-notes)
 
 ### New features and major changes
 
+- **Added the missing perfect groups of order up to a million**
+
+  Added perfect groups of orders that had been missing in the Holt/Plesken book
+  (newly computed).
+  This increases the number of groups in the perfect groups library by a factor
+  close to 6.
+  Also added five groups that were found missing in the existing lists.
+  See PR [#3925](https://github.com/gap-system/gap/pull/3925) for details.
```

Comment:
This is in 4.12.0, but not in 4.11.1

Reply:
For a patch release, new entries under "New features and major changes" seems fine and even desirable to me, I'd just hide that section here.

Reply:
O.K.